public/pacman-canvas.js

```diff
@@ -1469,22 +1469,18 @@ function checkAppCache() {
         // Mobile Control Buttons
         $(document).on('touchend mousedown','#up',function(event) {
             event.preventDefault();
-            window.navigator.vibrate(200);
```

Comment:
Does this effectively remove the vibration feedback when using the navigation buttons on Android devices? I think the vibration is cool feedback when using the buttons to control pacman and enhances the gaming experience. Do you know if there's an equivalent vibration call to enable this for iOS devices? If not, could we check if we're running on an Android device to invoke the vibrate function call?

Reply:
You're correct, it removes the vibration feedback when using the navigation buttons. 

I just sent a new commit including your suggestion, now it detects if the browser supports vibration before calling the vibrate function.

It seems that iOS doesn't support this feature: 
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/vibrate

Now it will vibrate on Android when supported by the browser.

Reply:
Awesome, nice work!